webrtc.html

```diff
@@ -1816,9 +1795,20 @@
                         </ol>
                       </li>
                       <li>
+                        <p>If <code><var>candidate</var>.ufrag</code> is not
+                        equal to any ufrag present in the corresponding
+                        <a>media description</a> of an applied remote
+                        description, reject <var>p</var> with an
```

Comment:
Ok. Then the logical conclusion is that you can have at most 2 ICE generations at the same time (during the time between when the offer is sent and the answer accepts the ICE restart). Then when the answer is applied, the old ICE checklists can be discarded. Does this need to be detailed in draft-ietf-mmusic-ice-sip-sdp? I initially was thinking JSEP, but you're allowed to reject an offer even with normal SDP offer/answer semantics, so this seems like a general "ICE SDP" thing.